cl/citations/management/commands/import_citations_csv.py

```diff
@@ -110,19 +117,25 @@ def add_arguments(self, parser):
             help="Limit number of rows to process.",
             required=False,
         )
+        parser.add_argument(
+            "--delay",
+            type=float,
+            default=1.0,
+            help="How long to wait to add each citation (in seconds, allows floating numbers).",
+        )
 
     def handle(self, *args, **options):
         if options["start_row"] and options["end_row"]:
             if options["start_row"] > options["end_row"]:
-                print("--start-row can't be greater than --end-row")
+                logger.info("--start-row can't be greater than --end-row")
```

Comment:
Again, if you set the default to zero, this gets simpler:

```
if options["start_row"] > options["end_row"]:
    raise ValueError("--start-row can't be greater than --end-row")

Reply:
Maybe something like this would be better:` if options["end_row"] and options["start_row"] > options["end_row"]:`

Because if you only pass start-row > 0, the default end-row would be 0 (or no limit) it will display the message because start-row > 0

Reply:
Sure.